### PR TITLE
Linux instructions + Dockerfile [wip]

### PR DESCRIPTION
**Pull request by Skrypt**
This is a dockerfile that will automate the creation of a Docker container with Orchard 2 and that will start "dotnet run" under Ubuntu 14.04 LTS. This is currently unstable due to the constant changes in the Core framework but it should work fine eventually.

### CODE REVIEW

**Comment by Jetski5822** [comment]
Could we add something to the readme file too?

**Comment by Skrypt** [comment]
Readme or in the wiki ?

**Comment by Skrypt** [comment]
Here it is I updated the readme.md file. Please review :smile:

File: README.md [hunk 1]
```diff
@@ -177,6 +177,90 @@ public void GetMePlacement()
...
+    sudo apt-get install git
+    git clone https://github.com/OrchardCMS/Orchard2.git
+
+    cd Orchard2
+    sh build.sh //really important to not do "sudo" else we will make the nuget packages to be accessible by only sudo wich will cause problems with the Omnisharp installer later on.
+    ```
+
+6. Install Omnisharp
+
+    Download with browser
```
Comment (sebastienros) [inline README.md]: What about using a `curl` command instead?

Comment (Skrypt) [inline README.md]: I tried and sometimes the .vsix file was corrupted because it would not download it completely. But I could retry to make sure.

**Comment by Jetski5822** [comment]
Good to get this in :) - even if we need to tweek it later

**Pull request merged**